bin/check-junit-upload.js

```diff
@@ -44,12 +47,13 @@ async function checkJunitUpload () {
     }
   }
   if (isSuccess) {
-    console.log(`✅ Successful check: the API returned ${data.length} tests.`)
+    console.log(`✅ Successful check: the API returned ${data.length} ${testLevel}s.`)
     process.exit(0)
   } else {
-    console.log(`❌ Failed check: the API did not return any test for the given filter.`)
+    console.log(`❌ Failed check: the API did not return any ${testLevel}s for the given filter.`)
     process.exit(1)
   }
 }
 
-checkJunitUpload()
\ No newline at end of file
+checkJunitUpload("test", process.argv[2] || '')
+checkJunitUpload("session", process.argv[2] || '')
```

Comment:
are we actually using `testLevel` filter somewhere? It seems we only use for logs: could we add it to the `filterQuery`?

Reply:
I think it'd also be more readable to use an environment variable for `extraFilter` rather than + `process.argv[2]`. 

We could set an `env` option in the workflow file, something like `EXTRA_QUERY_FILTER`: 
```
env: 
  ...
  EXTRA_QUERY_FILTER: @key2:40
```
and then use 
```
checkJunitUpload("session", process.env.EXTRA_QUERY_FILTER)
```

Reply:
> are we actually using testLevel filter somewhere? It seems we only use for logs: could we add it to the filterQuery?
Nice catch, somehow forgot to do that!